tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/dashboards/At-Risk_Learners.yaml

```diff
@@ -1038,6 +1077,19 @@ position:
     - TABS-7McownEo06
     - TAB-s_n47Pk1I
     type: ROW
+  TAB-1sC34gQvt:
+    children:
+    - ROW-duj4-KVlBB
+    id: TAB-1sC34gQvt
+    meta:
+      defaultText: Tab title
+      placeholder: Tab title
+      text: Help
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-7McownEo06
+    type: TAB
```

Comment:
Remember to add the help markdown text here.

Reply:
It's actually added at line 875, this is just the tab label itself. This whole yaml model is super confusing. :/